server/tests/test_notebook_view.py

```diff
@@ -151,6 +151,72 @@ def test_tryit_view(client, fake_user, logged_in, iomd):
         assert len(response.redirect_chain) == 0
 
 
+def test_from_template_view_reject_get_request(client):
+
+    resp = client.get(reverse("from-template"))
+    assert resp.status_code == 405
+
+
+def test_from_template_view_reject_no_iomd(client):
+
+    resp = client.post(reverse("from-template"), {"iomd": "",},)
+    assert resp.status_code == 400
+    assert resp.content == b"Must specify iomd template"
+
+
+def test_from_template_view_file_too_big(client, settings):
+    settings.MAX_FILE_SIZE = 8
+    with tempfile.NamedTemporaryFile(mode="w+") as f1, tempfile.NamedTemporaryFile(mode="w+") as f2:
+        f1.write("0")
+        f1.seek(0)
+        f2.write("0" * 2 * settings.MAX_FILE_SIZE)
+        f2.seek(0)
+        resp = client.post(
+            reverse("from-template"),
+            {
+                "iomd": "Test IOMD",
+                "title": "Test title",
+                "filename": "tempFile",
+                "file1": open(f1.name),
+                "file2": open(f2.name),
+            },
+        )
+        assert resp.status_code == 400
+        real_file_name = f2.name[f2.name.rfind("/") + 1 :]
+        assert resp.content == bytes(
+            f"File {real_file_name} exceeds maximum file size {settings.MAX_FILE_SIZE}", "utf-8"
+        )
+
+
+def test_from_template_view_post_success(client):
+    with tempfile.NamedTemporaryFile(mode="w+") as f1, tempfile.NamedTemporaryFile(mode="w+") as f2:
+        f1.write("0000")
+        f1.seek(0)
+        f2.write("1111")
+        f2.seek(0)
+
+        resp = client.post(
+            reverse("from-template"),
+            {
+                "iomd": "Test IOMD",
+                "title": "Test title",
+                "file1": open(f1.name),
+                "file2": open(f2.name),
+            },
+        )
+        f1_name = f1.name[f1.name.rfind("/") + 1 :]
+        f2_name = f2.name[f2.name.rfind("/") + 1 :]
```

Comment:
Try just using `os.path.basename` here, a bit simpler and won't cause linter errors:

https://docs.python.org/3/library/os.path.html#os.path.basename

(in fact you can just inline that in the check below instead of calculating a temporary variable)